Finish permanent booking with post-roll (#3586)
SkylineCommunications/dataminer-docs: user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_update_timing_booking_silently.md

 }
}
```

> [!NOTE]

> > Since SRM Framework 1.2.37/2.0.2 <!-- RN40481 --> it's possible to finish a permanent booking while defining post-roll. To do so, the timing of that booking should be changed, by changing the end date to a time in the near future and setting the appropriate post-roll time.

Reviewer: > From version 1.2.37/2.0.2 of the SRM framework onwards<!-- RN40481 -->, if a permanent booking is changed so that it finishes at a specified time in the near future, a post-roll time can also be specified.

> > Finishing a permanent booking will not support defining any post-roll.

Reviewer: This sentence isn't entirely clear to me, as it seems to contradict what was said just above. Do you mean that you can only define a post-roll time if you finish a permanent booking by defining an end time for it, but not if you finish the booking in a different way?

> > Although the reservation time of contributing bookings will adapt accordingly, if appropriate, no post-roll will be set. When orchestration would be required, the timing can be changed for each of them separately.

Reviewer: ```suggestion
> Although the reservation time of contributing bookings will adapt accordingly if appropriate, no post-roll time will be set for the contributing bookings. In case orchestration is required, you will need to change the timing for each of them separately.
```
